composer.json

```diff
@@ -10,9 +10,15 @@
             "homepage": "https://driesvints.com"
         }
     ],
+    "repositories": [
```

Comment:
Note - I'll update the PR after the new feature/script is merged in at: https://github.com/blade-ui-kit/blade-icons/pull/144